etc/adminhtml/system.xml

```diff
@@ -17,7 +17,17 @@
                        showInStore="0">
                     <label>Base URLs</label>
                 </field>
+                <field id="magentoUrlPushBack"
```

Comment:
Please snake_case the configuration. Also add `canRestore="1"` and make sure the default value is added in `etc/config.xml`.